core/src/main/java/hudson/node_monitors/ClockMonitor.java

```diff
@@ -46,8 +49,19 @@ public ClockDifference getDifferenceFor(Computer c) {
         return DESCRIPTOR.get(c);
     }
 
-    @Extension
-    public static final AbstractNodeMonitorDescriptor<ClockDifference> DESCRIPTOR = new AbstractAsyncNodeMonitorDescriptor<ClockDifference>() {
+    /**
```

Comment:
Symbol names are given to the type, so for those extensions to get symbols descriptors need to be named type.